pennylane/measurements/classical_shadow.py

```diff
@@ -358,17 +359,6 @@ def __init__(self, *args, H, seed=None, k=1, **kwargs):
         super().__init__(*args, **kwargs)
 
     def process(self, qscript, device):
-        """Compute expectation values using classical shadows in a differentiable manner.
-
-        Please refer to :func:`~.pennylane.shadow_expval` for detailed documentation.
-
-        Args:
-            qscript (QuantumScript): the quantum script to be processed
-            device (Device): the device used to process the quantum script
-
-        Returns:
-            float: expectation value estimate.
-        """
```

Comment:
Because it is already explained in the docstring of the class.

I am not sure what makes more sense here:
1. Document the MP in the docstring of the class.
2. Document the MP in the docstring of the `process` method.
3. Document in both.

While writing this I've thought that maybe the best option is to document both, using a more detailed description in the `process` method.

Reply:
I will do this in a separate PR because I want to merge this before the feature freeze